packages/styleguide/src/components/ExpandableLink/ExpandableLinkCallout.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { useMemo, useEffect, useRef, useState } from 'react'
+import { css, keyframes } from 'glamor'
+import { mUp } from '../../theme/mediaQueries'
+import zIndex from '../../theme/zIndex'
+import { useColorContext } from '../Colors/useColorContext'
+import RawHtml from '../RawHtml'
+import { LinkIcon } from '../Icons'
+import { fontStyles } from '../Typography'
+import { ellipsize } from '../../lib/styleMixins'
+import { StateProps, DELAY } from './ExpandableLink'
+
+type Props = {
+  inNativeApp?: boolean
+  timeOutRef: React.MutableRefObject<NodeJS.Timeout>
+  expandedLink: StateProps
+  setExpandedLink: React.Dispatch<React.SetStateAction<StateProps>>
+}
+
+const ExpandableLinkP = ({ children, ...props }) => {
+  const [colorScheme] = useColorContext()
+  return (
+    <p {...props} {...styles.contentText} {...colorScheme.set('color', 'text')}>
```

Comment:
```suggestion
    <p {...props} {...styles.contentText} {...Interaction.fontRule} {...colorScheme.set('color', 'text')}>
```

Currently faux bold and italic is rendered. By adding [`Interaction.fontRule`](https://styleguide.republik.ch/typography#bold-and-cursive) (needs to be imported as `interactionFontRule` in styleguide) we can fix this and also support `b` nested in `i` etc.

Reply:
We shouldn't support any styling / bold / italic in the callout. It should only render plain text.

Reply:
Well it was implemented with html support like we discussed when specing:
[
<img width="512" alt="Screenshot 2022-07-26 at 10 45 59" src="https://user-images.githubusercontent.com/410211/180964368-c40b838a-1e9c-4517-a8f6-d055850655b6.png">
](https://www.notion.so/republikmagazin/Inline-Link-Meta-Info-a3be338ba0bc4ea29a68187f78bea4f0)

italic and sub are necessary and very common – use for foreign words and carbon dioxide. bold and sup might not strictly be necessary – adjust the docs to only mention italic and sub?